databases/week1/week1.sql

```diff
@@ -0,0 +1,48 @@
+show DATABASES;
+use my_database_week1;
+SELECT * 
+from task
+SELECT COUNT (id) from task
+SELECT COUNT(id) from task
+ where due_date is NULL
+
+
+SELECT task.id, title, description, created, updated,due_date, user_id, status.name
+from task
+JOIN status ON task.status_id = status.id 
+where status.name= 'Done' 
```

Comment:
Here you can use `task.*` to select all fields from the `task` table. That way you do not have to write all of them out.